libs/dh/metering-point/feature-identity-and-master-data/src/lib/identity/dh-metering-point-identity.component.html

```diff
@@ -14,42 +14,73 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<div
-  class="overview__header"
-  *transloco="let t; read: 'meteringPoint.overview'"
->
-  <div class="watt-text-s">
-    <span [title]="t('meteringPointType')">{{
-      translationKeys?.meteringPointType | transloco | dhEmDashFallback
-    }}</span>
-
-    (<span [title]="t('meteringMethod')">{{
-      translationKeys?.meteringMethod | transloco | dhEmDashFallback
-    }}</span
-    >)
-  </div>
-
-  <h1 [title]="t('gsrnNumber')">{{ meteringPoint?.gsrnNumber }}</h1>
-
-  <div class="watt-text-s">
-    <span [title]="t('connectionState')">
+<div class="overview-header" *transloco="let t; read: 'meteringPoint.overview'">
+  <div class="watt-text-s overview-header__row">
+    <span [title]="t('connectionState')" class="watt-space-inline-xs">
       <dh-status-badge
-        [connectionState]="meteringPoint?.connectionState"
+        [connectionState]="identityDetails.connectionState"
       ></dh-status-badge>
     </span>
 
-    <ng-container *ngIf="meteringPoint?.settlementMethod">
-      |
-      <span [title]="t('settlementMethod')">
-        {{ translationKeys?.settlementMethod | transloco }}
-      </span>
-    </ng-container>
-
-    <ng-container *ngIf="meteringPoint?.readingOccurrence">
-      |
-      <span [title]="t('readingOccurrence')">
-        {{ translationKeys?.readingOccurrence | transloco }}
-      </span>
-    </ng-container>
+    <span [title]="t('meteringMethod')">
+      {{ translationKeys?.meteringPointType | transloco | dhEmDashFallback }}
+      ({{ translationKeys?.meteringMethod | transloco | dhEmDashFallback }})
+    </span>
+    |
+
+    <span
+      *ngIf="identityDetails?.settlementMethod"
+      [title]="t('settlementMethod')"
+    >
+      {{ translationKeys?.settlementMethod | transloco }}
+    </span>
+    |
+
+    <span [title]="t('readingOccurrence')">
+      {{ translationKeys?.readingOccurrence | transloco | dhEmDashFallback }}
+    </span>
+    |
```

Comment:
issue: the pipe is now outside of the `span`s but the way it's set up now, the user might end up seeing two pipes `| |` right next to each other in case `identityDetails?.settlementMethod` is `undefined` for some reason.

suggestion: for values that might not always be there, wrap the `span` and the related pipe inside an `ng-container` and move the `ngIf` there. As an example, see the deleted section right before this newly added one.